test/tools/schema_validator/test/schema_validator_test.cc

```diff
@@ -0,0 +1,103 @@
+#include "test/test_common/environment.h"
+#include "test/tools/schema_validator/validator.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
+class SchemaValidatorTest : public testing::Test {
+public:
+  void run(const std::string& command_line, const std::string& config_file) {
+    const std::string final_command_line = TestEnvironment::runfilesPath(
+        absl::StrCat("test/tools/schema_validator/test/config/", config_file));
+    const std::vector<std::string> split_command_line =
+        absl::StrSplit(fmt::format(command_line, final_command_line), ' ');
```

Comment:
Kind of unreliable in general to split on space, given " escaping and so on, but probably fine for these tests. Might be worth a comment.